documentation/puppetdb_connection.markdown

```diff
@@ -70,3 +81,32 @@ The default value is false.
 
 Hostname and port of the PuppetDB instance. `server_urls` takes precedence if both are defined.
 
+### High Availability configuration
+
+These settings are designed for use in a high-availability deployment of PuppetDB, available in Puppet Enterprise. While they may appear to work in other configurations, we discourage their use. In particular, absent the synchronization support in Puppet Enterprise, multiple PuppetDB instances can easily diverge from each other due to normal transient network issues. See the [PuppetDB HA Configuration Guide][ha_guide] for detailed configuration information.
```

Comment:
https://www.youtube.com/watch?v=E09LU6XVyxs maybe just s/we discourage their use/their use is discouraged/